Localización al Español

fgilio opened the pull request:
Con la colaboración de @gaby2300, de acuerdo a la Actividad: Instalar test pilot https://github.com/mozillahispano/meta/issues/5

deimidis commented on _pages/es/experiments/test-pilot.md:
```diff
@@ -10,53 +10,53 @@ parent: experiments
 ![image](/asserts/img/test-pilot.png)
 
-When building features for hundreds of millions of Firefox users worldwide, it’s important to get them right. To help figure out which features should ship and how they should work, we created the new Test Pilot program. Test Pilot is a way for you to try out experimental features and let us know what you think. You can turn them on and off at any time, and you’ll always know what information you ...
+Al momento de desarrollar características para los cientos de millones de usuarios de Firefox alrededor del mundo, es importante acertar. Para facilitar la decisión de qué funciones lanzar y cómo deberían funcionar, creamos el nuevo programa Test Pilot. El mismo permite a los usuarios probar características experimentales y a nosotros conocer su opinión. Puedes activarlas o desactivarlas en cualq ...
```
Gracias @fgilio y Gabriela. Tengo dos comentarios para este párrafo: 1) la primera frase queda medio rara, quizás sacándola del pasivo del inglés funcione mejor. 2) está muy *argentino* y debería ser más neutro.

fgilio replied:
Ok!

deimidis commented on _pages/es/experiments/test-pilot.md:
```diff
@@ -10,53 +10,53 @@ parent: experiments
+[Más información...](https://blog.mozilla.org/blog/2016/05/10/you-can-help-build-the-future-of-firefox-with-the-new-test-pilot-program/)
 
-## Impact and Metrics
+## Impacto y Métricas
```
Sacaría la mayúsculas en Métricas

fgilio replied:
Lo probamos

deimidis commented on _pages/es/experiments/test-pilot.md:
```diff
@@ -10,53 +10,53 @@ parent: experiments
+## Impacto y Métricas
 
-Firefox is key to how Mozilla expresses its values and makes the Web increasingly exciting in the face of walled gardens like individual apps. By testing new features and functionality, you help to keep Firefox on the leading edge and consumers engaged in our product. Additionally, it gives you a chance to go even deeper with specific feedback and ideas for Firefox.
+Firefox es clave en cómo Mozilla expresa sus valores y hace la Web cada vez más emocionante detrás de jardines amurallados, como las aplicaciones individuales.
```
Recuerden que en una *localización* nos podemos tomar más libertades que la traducción. Me parece que está muy literal

deimidis replied:
Al agregar un párrafo Github se volvió loco :)

fgilio replied:
Si, está medio raro así. Le podemos pegar una vuelta de rosca

fgilio commented:
@deimidis hicimos algunas correcciones

nukeador commented:
@deimidis is this OK to merge now?

nukeador commented:
There are some conflicts to merge now. Changes were made to the original file. 

https://github.com/mozilla/activate.mozilla.community/commits/gh-pages/_pages/es/experiments/test-pilot.md

gaby2300 commented:
Do we we have to translate the changes then? If so, could you please tell
us how/where can we see them?



On 26 September 2016 at 18:24, Nukeador <notifications@github.com> wrote:

> There are some conflicts to merge now. Changes were made to the original
> file.
>
> https://github.com/mozilla/activate.mozilla.community/
> commits/gh-pages/_pages/es/experiments/test-pilot.md
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/activate.mozilla.community/pull/61#issuecomment-249701902>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACDVaL9JZc9bvKJX9vlGbp8BQAEyqfU-ks5quDf7gaJpZM4J7rlx>
> .
>

nukeador commented:
Changes are incompatible and need manual solution.

Using git merge and manual pick is one solution, but my recommendation for @fgilio is to get the actual page, apply the text changes and send a new PR.

Sorry for this, it we took too long to integrate it into the main branch.

gaby2300 commented:
Right, no problem @nukeador.

I'll contact @fgilio then, thanks!



On 26 September 2016 at 20:14, Nukeador <notifications@github.com> wrote:

> Changes are incompatible and need manual solution.
>
> Using git merge and manual pick is one solution, but my recommendation for
> @fgilio <https://github.com/fgilio> is to get the actual page, apply the
> text changes and send a new PR.
>
> Sorry for this, it we took too long to integrate it into the main branch.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/activate.mozilla.community/pull/61#issuecomment-249724631>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACDVaFrlg_U3ua66tucDdUGbkrP5z8Rzks5quFHMgaJpZM4J7rlx>
> .
>

fgilio commented:
@nukeador i'll make the corrections and check them with @gaby2300 before committing the update

fgilio commented:
@nukeador it's done, changes where really really small

nukeador commented:
I still see there is a conflict and we are unable to auto merge. Maybe you can open a new fresh PR?

fgilio commented:
@nukeador it has no conflicts now: https://github.com/mozilla/activate.mozilla.community/pull/75

Maybe it was some empty lines that we removed on this pr, but I've added those again in the new one
